ext/opentelemetry-ext-wsgi/README.rst

```diff
@@ -1,9 +1,22 @@
 OpenTelemetry WSGI Middleware
 =============================
 
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-wsgi.svg
```

Comment:
Can't we just use `image::` without the `|pypi|` stuff?

Reply:
I think this syntax is more readable when there are many badges, for instance:

```
|GitterChat| |BuildStatus| |PyPI| |ReadTheDocs|
```